data/mods/gen2/moves.js

```diff
@@ -852,8 +852,8 @@ let BattleMovedex = {
 					move.secondaries = move.secondaries.filter(p => !p.kingsrock);
 				}
 				if (move.drain) {
-					this.add('-hint', "In Gold/Silver/Crystal, draining moves always miss against Substitute.");
 					this.add('-miss', source);
+					this.hint("In Gen 1, draining moves always miss against Substitute.");
```

Comment:
...this is Gen 2.

Reply:
```suggestion
					this.hint("In Gen 2, draining moves always miss against Substitute.");
```

?

I don't know the actual mechanic, but in case it is this simple, clicking "Accept change" is probably easier for you than fixing this yourself.